Can't use `libraryGenerator` from `@nrwl/workspace` without `@angular-devkit/schematics`

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior

I'm trying to create a custom generator that extends `libraryGenerator` from `@nrwl/workspace`, but I get an error when I try to run it without adding `@angular-devkit/schematics` first.

## Expected Behavior
<!-- What is the behavior that you expect to happen? -->
<!-- Is this a regression? .i.e Did this used to be the behavior at one point?  -->

I expect to be able to use the Nx's library generator without adding `@angular-devkit/schematics` as a dependency. It's not mentioned anywhere in the [documentation](https://nx.dev/generators/composing-generators#using-nx-devkit-generators) on composing generators (the documentation is even using this specific generator).

## Steps to Reproduce
<!-- Help us help you by making it easy for us to reproduce your issue! -->

<!-- Can you reproduce this on https://github.com/nrwl/nx-examples? -->
<!-- If so, open a PR with your changes and link it below. -->
<!-- If not, please provide a minimal Github repo -->
<!-- At the very least, provide as much detail as possible to help us reproduce the issue -->

### Failure Logs
<!-- Please include any relevant log snippets or files here. -->
```shell
Cannot find module '@angular-devkit/schematics'
Require stack:
- /Users/my-user/frontend/node_modules/.pnpm/@nrwl+workspace@14.1.5_y6u32vedbwjta2g3ygg56bhflu/node_modules/@nrwl/workspace/src/utils/ast-utils.js
- /Users/my-user/frontend/node_modules/.pnpm/@nrwl+workspace@14.1.5_y6u32vedbwjta2g3ygg56bhflu/node_modules/@nrwl/workspace/src/utils/project-type.js
- /Users/my-user/frontend/node_modules/.pnpm/@nrwl+workspace@14.1.5_y6u32vedbwjta2g3ygg56bhflu/node_modules/@nrwl/workspace/index.js
- /Users/my-user/frontend/dist/tools/generators/make-lib/index.js
- /Users/my-user/frontend/node_modules/.pnpm/nx@14.1.5_typescript@4.6.4/node_modules/nx/src/config/workspaces.js
- /Users/my-user/frontend/node_modules/.pnpm/nx@14.1.5_typescript@4.6.4/node_modules/nx/src/command-line/generate.js
- /Users/my-user/frontend/node_modules/.pnpm/nx@14.1.5_typescript@4.6.4/node_modules/nx/src/command-line/workspace-generators.js
- /Users/my-user/frontend/node_modules/.pnpm/nx@14.1.5_typescript@4.6.4/node_modules/nx/src/command-line/nx-commands.js
- /Users/my-user/frontend/node_modules/.pnpm/nx@14.1.5_typescript@4.6.4/node_modules/nx/bin/init-local.js
- /Users/my-user/frontend/node_modules/.pnpm/nx@14.1.5_typescript@4.6.4/node_modules/nx/bin/nx.js
 ELIFECYCLE  Command failed with exit code 1.
```

### Environment
<!-- It's important for us to know the context in which you experience this behavior! -->
<!-- Please paste the result of `nx report` below! -->
   Node : 16.13.0
   OS   : darwin x64
   pnpm : 7.1.2
   
   nx : 14.1.5
   @nrwl/angular : Not Found
   @nrwl/cypress : 14.1.5
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.1.5
   @nrwl/eslint-plugin-nx : 14.1.5
   @nrwl/express : Not Found
   @nrwl/jest : 14.1.5
   @nrwl/js : Not Found
   @nrwl/linter : 14.1.5
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : 14.0.3
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : Not Found
   @nrwl/web : Not Found
   @nrwl/workspace : 14.1.5
   typescript : 4.6.4
   rxjs : 6.6.7
   ---------------------------------------
   Community plugins:
         @nxext/vite: 14.0.0

--- a/docs/shared/generators/composing-generators.md
+++ b/docs/shared/generators/composing-generators.md
@@ -7,7 +7,7 @@ Generators are useful individually, but reusing and composing generators allows 
 Nx Devkit generators can be imported and invoked like any javascript function. They often return a `Promise`, so they can be used with the `await` keyword to mimic synchronous code. Because this is standard javascript, control flow logic can be adjusted with `if` blocks and `for` loops as usual.
 
 ```typescript
-import { libraryGenerator } from '@nrwl/workspace';
+import { libraryGenerator } from '@nrwl/workspace/generators';
 
 export default async function (tree: Tree, schema: any) {
   await libraryGenerator(

--- a/docs/shared/generators/creating-files.md
+++ b/docs/shared/generators/creating-files.md
@@ -41,7 +41,7 @@ import {
   joinPathFragments,
   readProjectConfiguration,
 } from '@nrwl/devkit';
-import { libraryGenerator } from '@nrwl/workspace';
+import { libraryGenerator } from '@nrwl/workspace/generators';
 
 export default async function (tree: Tree, schema: any) {
   await libraryGenerator(tree, { name: schema.name });

--- a/docs/shared/generators/generator-options.md
+++ b/docs/shared/generators/generator-options.md
@@ -15,7 +15,7 @@ Import the TypeScript schema into your generator file and replace the any in you
 
 ```typescript
 import { Tree, formatFiles, installPackagesTask } from '@nrwl/devkit';
-import { libraryGenerator } from '@nrwl/workspace';
+import { libraryGenerator } from '@nrwl/workspace/generators';
 
 export default async function (tree: Tree, schema: GeneratorOptions) {
   await libraryGenerator(tree, { name: `${schema.name}-${schema.type || ''}` });

--- a/docs/shared/generators/workspace-generators.md
+++ b/docs/shared/generators/workspace-generators.md
@@ -35,7 +35,7 @@ The initial generator function creates a library.
 
 ```typescript
 import { Tree, formatFiles, installPackagesTask } from '@nrwl/devkit';
-import { libraryGenerator } from '@nrwl/workspace';
+import { libraryGenerator } from '@nrwl/workspace/generators';
 
 export default async function (tree: Tree, schema: any) {
   await libraryGenerator(tree, { name: schema.name });

